conf/application.conf

```diff
@@ -178,6 +178,13 @@ akka {
   }
 }
 
+stats {
+    inprogressfiles {
+        initialdelay = 60000 milliseconds
+        interval = 60000 milliseconds
```

Comment:
Could we change these from "60000 milliseconds" to "1 minute" so that my slow brain doesn't have to do maths? :)

Reply:
As discussed, proper support for Duration from play.api.Configuration (configuration.get[Duration](....) ) came in Play 2.6, I think.